### DISCUSSION TITLE
Add BlockWallet Connector

### DISCUSSION THREAD

**Discussion by julianariel** [wevm/wagmi#1364] [Ideas]
Hey,

I'm Julián from the **BlockWallet** team and I'd like to propose the addition of our wallet to wagmi.

**BlockWallet** is a browser extension, privacy-first, self-custodial web3 wallet that works with every EVM-compatible network. Our product has a bunch of integrated features like in-wallet swaps, bridges, flashbots integration and many more.

You can check this with more detail on our socials:
- https://blockwallet.io/
- https://twitter.com/getblockwallet

Please let me know if we can help or contribute to achieving this.

Thanks!

<p align="center">
  <a href="https://blockwallet.io">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/11839151/188500975-8cd95d07-c419-48aa-bb85-4200a6526f68.svg" />
      <source media="(prefers-color-scheme: light)" srcset="https://blockwallet.io/static/images/logo-blockwallet-black.svg" />
      <img src="[https://blockwallet.io/static/images/logo-medium.svg](https://user-images.githubusercontent.com/11839151/188500975-8cd95d07-c419-48aa-bb85-4200a6526f68.svg)" width="200" />
    </picture>
  </a>
</p>

**Comment by tmm** [comment]
We aren't adding additional connectors to the core right now as we are getting ready to standardize this process. In the meantime, you can package it up as a standalone package. You are also welcome to submit a PR to include a custom name for BlockWallet in the `InjectedConnector`.

**Comment by julianariel** [comment]
@tmm I'll submit the PR soon then, thanks!
